pkg/image/export.go

```diff
@@ -15,6 +15,8 @@ import (
 func ToBasicJSON(image *models.Image) *jsonschema.Image {
 	newImageJSON := jsonschema.Image{
 		Title:     image.Title,
+		URL:       image.URL,
+		Date:      image.Date.Time.String(),
```

Comment:
This will panic if date is `nil`. See other `Date` export examples.

Reply:
Done